src/Umbraco.Core/PropertyEditors/PropertyEditor.cs

```diff
@@ -13,6 +14,7 @@ namespace Umbraco.Core.PropertyEditors
     /// <remarks>
     /// The Json serialization attributes are required for manifest property editors to work
     /// </remarks>
+    [DebuggerDisplay("{DebuggerDisplay(),nq}")]

```

Comment:
Nq is "no quotes". http://blogs.msdn.com/b/jaredpar/archive/2011/03/18/debuggerdisplay-attribute-best-practices.aspx